multisense_ros/src/camera.cpp

```diff
@@ -212,6 +217,257 @@ cv::Vec3b u_interpolate_color(double u, double v, const cv::Mat &image)
     return result;
 }
 
+Eigen::Matrix<uint8_t, 3, 1> groundSurfaceClassToPixelColor(const uint8_t value)
```

Comment:
Maybe we should move some of the functions to a `spline.cc` or `groundsurface.cc` utility